department-of-veterans-affairs/vets-website
# Pension 73987 disclosure title

## Summary
Update disclosure titles on the Pension Benefits form (21P-527EZ)

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Go to the dependents section of the 'Household information' chapter
3. Verify that the legend is displayed with the correct information
4. Verify that the screen reader includes the dependents name when announcing the legend on the subsequent dependent pages

## Acceptance criteria
[Issue in Github - 74871](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73987)

## Related issue(s)
n/a

## Testing done

## Screenshots
| Before | After |
| ------ | ------ |
|![localhost_3001_pension_application_527EZ_ (3)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/97c8f17e-dbba-42cc-87e6-4c07cd8de077)|![localhost_3001_pension_application_527EZ_review-and-submit (5)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/27e1ffc2-6034-4219-988b-d36e85694af5)

## What areas of the site does it impact?
Pension Benefits Application

### Quality Assurance & Testing

### Error Handling


### Authentication
